packages/cli/generators/model/index.js

```diff
@@ -154,8 +164,30 @@ module.exports = class ModelGenerator extends ArtifactGenerator {
       },
     ])
       .then(props => {
-        if (typeof props.modelBaseClass === 'object')
-          props.modelBaseClass = props.modelBaseClass.value;
+        if (!this.isBaseClassChecked) {
```

Comment:
Nitpick: use reverse logic and early return to reduce indentation level and make the code easier to read.

```suggestion
        if (this.isBaseClassChecked) return;
```